index.html

```diff
@@ -69,13 +69,13 @@ <h1>I'm Zermarden <br>
                     <span>CANOPY</span>
```

Comment:
- [ ] I kindly suggest that you make use of an `h2` to `h6` for your project header named `Topic` because it is a header.